### PR TITLE
Creating a Django Environment

### PR DESCRIPTION
**Pull request by LiorKGOW**
Installing Django using Pipenv and setting up a django environment for the project.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -39,7 +39,7 @@ Vagrant.configure("2") do |config|
   # your network.
   # config.vm.network "public_network"
   config.vm.network(
-	"forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
+	  "forwarded_port", guest: 8000, host: 8000, host_ip: "127.0.0.1"
   )
```
Comment (adiabramovitch) [inline Vagrantfile:43]: There is a redundant space

**Review by ronyyosef — COMMENTED** [review]
Great work @LiorKGOW

File: Vagrantfile (hunk 1 above)
Comment (LiorKGOW) [reply Vagrantfile]: Fixed

**Comment by ShellyGolden** [comment]
looks good @LiorKGOW

**Pull request merged**